src/sourmash/sig/__main__.py

```diff
@@ -1283,6 +1291,98 @@ def fileinfo(args):
         print(json.dumps(info_d))
 
 
+def check(args):
+    """
+    check signature db(s) against a picklist.
+    """
+    from sourmash.picklist import PickStyle
+    set_quiet(args.quiet)
+    moltype = sourmash_args.calculate_moltype(args)
+    picklist = sourmash_args.load_picklist(args)
+    pattern_search = sourmash_args.load_include_exclude_db_patterns(args)
+    _extend_signatures_with_from_file(args)
+
+    if not picklist:
+        error("** No picklist provided?! Exiting.")
+        sys.exit(-1)
```

Comment:
not allowing ksize/name selection alone here? I guess we already have that functionality in `sig extract` and `sig grep`, eh?

Reply:
correct - `check` is all about checking against a picklist, and, in fact, does not actually output signatures at all!